### DISCUSSION TITLE
How to highlight selection with tab?

### DISCUSSION THREAD

**Discussion by ChillarAnand** [marlonrichert/zsh-autocomplete#516] [Q&A]
When cycling through completions, using down arrow highlights the selection. But tab doesn't highlight.

With down arrow.

<img width="627" alt="Screenshot 2022-11-15 at 18 28 41" src="https://user-images.githubusercontent.com/4463796/201925888-f8b6af6f-129b-49f7-a29d-fe169c3edd6d.png">

With tab.

<img width="627" alt="Screenshot 2022-11-15 at 18 29 03" src="https://user-images.githubusercontent.com/4463796/201925898-e54e565c-2f21-4e31-9543-ec19cd2c0b2b.png">

How to highlight with tab as well?

**Comment by marlonrichert** [comment]
By default, Tab simply inserts the top completion. It does not enter the menu

To change this, add to your `.zshrc` file, but change the last word to `menu-select`:

https://github.com/marlonrichert/zsh-autocomplete/blob/f07efda31f27446c8940b069d054bb8b0f61e026/.zshrc#L61

Documentation can be found in the file above.

**Reply by ChillarAnand** [reply]
I am using zsh plugins to include autocomplete in plugins list. I have added that line above/below plugins list. But it doesn't seem to work. @marlonrichert

**Reply by marlonrichert** [reply]
Can you show me your `.zshrc` file?

**Reply by ChillarAnand** [reply]
`.zshrc` file is available here. https://github.com/ChillarAnand/init/blob/main/zshrc.sh

**Reply by marlonrichert** [reply]
You are overriding it on this line:

https://github.com/ChillarAnand/init/blob/98e155a94db9a41a5857bf517b64b0c448e719c6/zshrc.sh#L132

Oh, and my bad: You don't have to put it before sourcing Autocomplete. It also works after. It just doesn't work at runtime.
